OWNERS.md

```diff
@@ -1,22 +1,51 @@
 * See [CONTRIBUTING.md](CONTRIBUTING.md) for general contribution guidelines.
-* See [GOVERNANCE.md](GOVERNANCE.md) for governance guidelines.
+* See [GOVERNANCE.md](GOVERNANCE.md) for governance guidelines and maintainer responsibilities.
+
+This page lists all active maintainers and their areas of expertise. This can be used for
+routing PRs, questions, etc. to the right place.
 
 # Senior maintainers
 
 * Matt Klein ([mattklein123](https://github.com/mattklein123)) (mklein@lyft.com)
+  * Catch-all, "all the things", and generally trying to make himself obsolete as fast as
+    possible.
 * Harvey Tuch ([htuch](https://github.com/htuch)) (htuch@google.com)
+  * APIs, xDS, gRPC, configuration, and Bazel/build.
```

Comment:
I'm also happy to cover some of the same topics that @dnoe is expert in, since I've been involved to some extent in either hacking around with or reviewing a lot of these. Also, I can tackle Python and bash fu.